blog/urls.py

```diff
@@ -5,5 +5,5 @@
 	url(r'^$', views.post_list, name='post_list'),
 	url(r'^post/(?P<pk>[0-9]+)/$', views.post_detail, name='post_detail'),
 	url(r'^post/new/$', views.post_new, name='post_new'),
-	url(r'^post/(?P<pk>[0-9]+)/edit/$', views.post_edit, name='post-edit'),
-]
\ No newline at end of file
+	url(r'^post/(?P<pk>[0-9]+)/edit/$', views.post_edit, name='post_edit'),
```

Comment:
Your post detail template and the rest of URLs use a snake_case instead of hyphen-case. Changing fixes an error that pops up.